### PR TITLE
Initial content for demo page

### PR DESCRIPTION
**Pull request by svrnm**
This is giving the demo project a home within the website. Documentation can grow from here.

cc @open-telemetry/demo-maintainers , @open-telemetry/demo-approvers 

---

Preview: https://deploy-preview-1826--opentelemetry.netlify.app/community/demo/

### CODE REVIEW

File: content/en/community/demo/_index.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+title: OpenTelemetry Demo
```
Comment (chalin) [inline content/en/community/demo/_index.md:2]:
```suggestion
title: OpenTelemetry Demo
linkTitle: Demo
```

**Comment by chalin** [comment]
The permissions issues reported by GH:

> image

are now fixed:

> image

File: content/en/community/demo/_index.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+---
+
+{{% alert title="Note" color="warning" %}} The demo is a work in progress. If
```
Comment (chalin) [inline content/en/community/demo/_index.md:5]:
```suggestion
{{% alert title="Important" color="warning" %}} The demo is a work in progress. If
```

**Review by chalin — APPROVED** [review]

**Review by reyang — APPROVED** [review]

**Comment by melbgirl** [comment]
This is great: we'll be able to see more information about how to get involved in the OTel community (and how to contribute) without needing to dip into GitHub readme's and gather the information through various pages. It should really help potential contributors more easily understand how to participate.

**Review by cartermp — APPROVED** [review]

**Review by cartersocha — APPROVED** [review]

**Review by julianocosta89 — APPROVED** [review]

**Pull request merged**